File: Libro.java
import java.util.Random;

public class Libro {
    Random ran = new Random();
    private String title, author;
    private int id;
    private boolean rented = false;

    public Libro(String title, String author){
        this.id = ran.nextInt(1,1000);
        this.title = title;
        this.author = author;
    }

    public String getTitle() {
        return title;
    }

    public String getAuthor() {
        return author;
    }

    public int getId() {
        return id;
    }

    public boolean isRented() {
        return rented;
    }

    public void setRented(boolean rented) {
        this.rented = rented;
    }
}

File: libro.py
import random


class Libro:
    __id = 0
    __title = ""
    __author = ""
    __rented = False

    def __init__(self, name, author):
        self.__id = random.randint(1, 1000)
        self.__title = name
        self.__author = author

    def get_title(self):
        return self.__title

    def get_author(self):
        return self.__author

    def get_id(self):
        return self.__id

    def is_rented(self):
        return self.__rented

    def set_rented(self, rented):
        self.__rented = rented
